src/codeu/chat/client/commandline/Chat.java

```diff
@@ -181,6 +181,20 @@ private UserContext findUser(String name) {
       }
     });
 
+    panel.register(“info”, new Panel.Command() {
+      @Override
+      public void invoke(Scanner args) {
+        final ServerInfo info = context.getInfo();
+        if (info == null) {
+          // Communicate error to user - the server did not send us a valid
```

Comment:
You'll need to log the error here, with something like 
`LOG.error(ex, "Exception during call on server.");`